src/index.html

```diff
@@ -5,8 +5,449 @@
+              data-qa="nav-hover"
+            >
+              laptops & computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#gadgets">
+              gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#tablets">
+              tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#photo">
+              photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#video">
+              video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="card" data-qa="card">
+        <img
+          src="images/imac.jpeg"
+          alt="APPLE A1419 iMac 27"
+          class="card__img"
+        >
+        </img>
+
+        <h2 class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__code">
+          Product code: 195434
+        </p>
+
+        <div class="card__review">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <div class="card__review-text">
+            Reviews: 5
+          </div>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-text">
+            Price:
+          </p>
+
+          <p class="card__price-money">
+            $2,199
+          </p>
+        </div>
+        <a href="#"
+          class="card__button"
+          data-qa="card-hover"
+        >
+          buy
+        </a>
+      </div>
+
+      <div class="card">
+        <img
+          src="images/imac.jpeg"
+          alt="APPLE A1419 iMac 27"
+          class="card__img"
+        >
+        </img>
+
+        <h2 class="card__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+
+        <p class="card__code">
+          Product code: 195434
+        </p>
+
+        <div class="card__review">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <div class="card__review-text">
+            Reviews: 5
+          </div>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-text">
+            Price:
+          </p>
+
+          <p class="card__price-money">
+            $2,199
+          </p>
+        </div>
+        <a href="#"
+          class="card__button"
+          data-qa="hover"
+        >
```

Comment:
Please fix everywhere
```suggestion
        <a 
          href="#"
          class="card__button"
          data-qa="hover"
        >
```